field-generator/Makefile

```diff
@@ -0,0 +1,46 @@
+# Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+target := cloud
+#target := local
+
+export VESPA_CLI_CLOUD_SYSTEM=public
+#export VESPA_CLI_CLOUD_SYSTEM=publiccd
```

Comment:
I'm not sure we want to leak references to our internal CI environment in sample apps.